client/src/components/History/CurrentHistory/HistoryFilters.vue

```diff
@@ -4,6 +4,7 @@
             <DebouncedInput v-model="localFilter" v-slot="{ value, input }">
                 <b-form-input
                     size="sm"
+                    :class="[inputBg]"
```

Comment:
No need for array parentheses ([). Suggest to just write something like `:class="filterText && 'has-text'"` here.

Reply:
Suggest also to directly use `border-info` and `bg-info` bootstrap classes, and avoid using custom classes and `$state-` colors.